tools/lint-docstring

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+
+import argparse
+import importlib
+import sys
+from os import getcwd, listdir
+from os.path import isfile, join
+from typing import Dict, List, Tuple, cast
+
+
+developer_doc = join(getcwd(), 'docs', 'developer-file-overview.md')
```

Comment:
`getcwd` won't always work for this, eg. if the script is run from in `tools/`. Consider using operations derived from `__file__`, perhaps?